README.md

```diff
@@ -73,14 +73,14 @@ To create 4-bit hashes for input data of 7 dimensions:
         storage_config={"dict":None}
     )
 
-    # If you're using >= v2.1.0, this is much faster
-    # lsh.index(X, extra_data=y)
+    lsh.index(X, extra_data=y)
 
-    # For all versions
-    for ix in range(X.shape[0]):
-        x = X.getrow(ix)
-        c = y[ix]
-        lsh.index( x, extra_data=c)
+    # If you're using NumPy < v2.1.0, this indexing
+    # procedure is compatible but much slower
+    #for ix in range(X.shape[0]):
+    #    x = X.getrow(ix)
+    #    c = y[ix]
+    #    lsh.index(x, extra_data=c)
```

Comment:
You changed the comment, making it incorrect. Please revert this change.

Reply:
Done.

Reply:
Can you revert this back to original? As it is in main currently. There's no such thing as Numpy v2

Reply:
Done. I though that by "using >= v2.1.0,", you meant NumPy >= v2.1.0.
I fixed it so that it is clear that the version of SparseLSH is specified.